eng/scripts/KillProcesses.ps1

```diff
@@ -22,6 +28,8 @@ _kill vctip.exe
 _kill chrome.exe
 _kill h2spec.exe
 _kill WerFault.exe
+_killSeleniumJavaInstances
```

Comment:
Why isn't this `_kill java.exe`? or a few `_kill` calls for the Java processes we use?

Reply:
To be clear: I won't block on this but it looks more complicated that necessary and will fail the build if a `Stop-Process` fails (unlike in the `_kill` function).

Reply:
I already fixed that locally. I'm trying it out on a different PR. I will update this one and merge it once  I see it working on the other one.